### ISSUE TITLE
Add validation of WORKING_DIR

### ISSUE THREAD

**Issue by hardys** [openshift-metal3/dev-scripts#895]
WORKING_DIR defaults to /opt/dev-scripts, which isn't ideal because if you install e.g RHEL8 with the default partition scheme, you only get 50G on the root LV.

A common solution to this is to override WORKING_DIR, but you must not put the directory inside an existing user home directory (because it's not world readable), if you do then the vbmc power control will fail with an error like:

```
Stderr: 'Running Get PICMG Properties my_addr 0x20, transit 0, target 0x20\nError response 0xc1 from Get PICMG Properities\nRunning Get VSO Capabilities my_addr 0x20, transit 0, target 0x20\nInvalid completion code received: Invalid command\nDiscovered IPMB address 0x0\nSet Chassis Power Control to Up/On failed: Node busy\n': oslo_concurrency.processutils.ProcessExecutionError: Unexpected error while running command.
```

We should add validation to check that $WORKING_DIR is world readable and/or isn't in any existing user homedir.

Probably some validation of the space available would also be good, either by figuring out a sensible limit or perhaps better by checking for free space in common.sh and bailing out if the volume backing $WORKING_DIR is 90% full or something?

**Comment by andfasano** [comment]
Validations check about the fact that the $WORKING_DIR is world readable and that enough space is available looks reasonable to me and very likely do not add any breaking change and should be sufficient. I would avoid checking if it's in the homedir.

**Comment by hardys** [comment]
Closed via #900
